pex/bin/pex.py

```diff
@@ -653,6 +653,12 @@ def _compatible_with_current_platform(interpreter, platforms):
 
 
 def main(args=None):
+    if "PEX3_CLI" in os.environ:
```

Comment:
Have you tried `PEX_SCRIPT=pex3 ./pex.pex`? `pex --help-variables` is invaluable.

Reply:
Doh. Thanks. 

`PEX_SCRIPT=pex3 python3 -m pex -h` doesn't work but that doesn't matter to me. `PEX_SCRIPT=pex3 dist/pex` does 🙌

Reply:
Re the 1st command:

PEX_ are Pex runtime variables, these take effect at Pex runtime, i.e.: when a Pex boots up.

`python -m module` uses Python to boot (via the runpy machinery) not Pex.